Python Part/network_models.py

```diff
@@ -1,15 +1,17 @@
 from lib.cnn import cnn_layers
 from lib.dnn_layers import dnn_layers
 from utils.image_helper import *
+from lib.fnn import fnn_layers
 
 
 class network_models:
-    def __init__(self, images_train, labels_train, images_test, labels_test, n_kernels,kernel_size, epoch, alpha, hidden):
+    def __init__(self, images_train, labels_train, images_test, labels_test, n_kernels, kernel_size, epoch, alpha,
+                 hidden):
         self.images_train = images_train
         self.labels_train = labels_train
         self.images_test = images_test
         self.labels_test = labels_test
-        self.cnn = cnn_layers(n_kernels=n_kernels,kernel_size=kernel_size)
+        self.cnn = cnn_layers(n_kernels=n_kernels, kernel_size=kernel_size)
```

Comment:
having something like this in our class (an instance of other class, inside our class) is considered a bad practice, because it'll increase the dependency between the classes (like adding wires), i.e: if we don't have cnn class, we won't be able to use the class unless we make some edits and remove it, because this class is DEPENDENT upon cnn. but let's just ignore it for the time being.